Calculate/Calculator.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Calculate
+{
+    public class Calculator
+    {
+
+        public static Func<int, int, int> Add = (int1, int2) => int1 + int2;
+        public static Func<int, int, int> Subtract = (int1, int2) => int1 - int2;
+        public static Func<int, int, int> Multiply = (int1, int2) => int1 * int2;
+        public static Func<int, int, int> Divide = (int1, int2) =>
+        {
+            return int2 == 0 ? throw new DivideByZeroException("Tried to divide by zero") : int1 / int2;
+        };
+
+        public static IReadOnlyDictionary<char, Func<int, int, int>> MathematicalOperations { get; } = new Dictionary<char, Func<int, int, int>>()
+        {
+            { '+', Add },
+            { '-', Subtract },
+            { '*', Multiply },
+            { '/', Divide }
+
+        };

```

Comment:
This is perfect! My only comment would be to multiline it for better readability: 

```suggestion
        public static IReadOnlyDictionary<char, Func<int, int, int>> MathematicalOperations { get; } = 
        new Dictionary<char, Func<int, int, int>>()
        {
            { '+', Add },
            { '-', Subtract },
            { '*', Multiply },
            { '/', Divide }

        };

```